cmake/FindCython.cmake

```diff
@@ -4,7 +4,7 @@
 
 # This finds the "cython" executable in your PATH, and then in some standard
 # paths:
-FIND_FILE(CYTHON_BIN cython /usr/bin /usr/local/bin)
+FIND_PROGRAM(CYTHON_BIN cython /usr/bin /usr/local/bin)
```

Comment:
I was wrong. Both are essentially the same, but the name we are searching for is `cython`. `find_file` sees `cython.exe` and ignores it while `find_program` finds it.